sdks/python/apache_beam/io/textio_test.py

```diff
@@ -504,6 +578,22 @@ def test_write_gzip_file_empty(self):
     with gzip.GzipFile(self.path, 'r') as f:
       self.assertEqual(f.read().splitlines(), [])
 
+  def test_write_text_file_with_header(self):
+    header = 'header1\nheader2'
+    sink = TextSink(self.path, header=header)
+    self._write_lines(sink, self.lines)
+
```

Comment:
Add a test that runs a pipepine that uses textio.WriteToText where header is not None.